app/views/stations/dashboard.erb

```diff
@@ -0,0 +1,65 @@
+  <h1>Stations Overview</h1>
+
+  <img class="img-responsive" src="images/bike_share_2.jpg">
+  <div>
+    <br>
+  </div>
+
+  <p><a class="btn btn-primary" href="/stations">All Stations</a></p>
+
+
+<table class="table table-striped table-hover">
+
+<tr>
+  <td><h3> Total Station Count</td></h3>
+<td><h3> <%= Station.count %> </td></h3>
```

Comment:
same here - we want our controller to get the data and our view to simply display it 👍